apps/io-public/src/utils/validation-token.ts

```diff
@@ -0,0 +1,37 @@
+import * as crypto from "crypto";
+
+import * as O from "fp-ts/lib/Option";
+import * as TE from "fp-ts/lib/TaskEither";
+
+import { TableClient } from "@azure/data-tables";
+import { ValidationTokenEntityAzureDataTables } from "@pagopa/io-functions-commons/dist/src/entities/validation_token";
+import {
+  retrieveTableEntityDecoded,
+  StorageError
+} from "../utils/azure_storage";
```

Comment:
Useless 'utils' path reference

```suggestion
import { retrieveTableEntityDecoded, StorageError } from "./azure_storage";
```

Reply:
thx, done in be5085b